core/src/main/scala/za/co/absa/spline/harvester/builder/read/NoOpReadRelationHandler.scala

```diff
@@ -0,0 +1,28 @@
+package za.co.absa.spline.harvester.builder.read
```

Comment:
The build is failing because the licence is missing in some files, please include the licences as you did in  ReadRelationHandler.scala